### DISCUSSION TITLE
Is there a way to auto group tab names when groupping tabs together in sideberry?

### DISCUSSION THREAD

**Discussion by RayShonami** [mbnuqw/sidebery#1726] [Q&A]
I want to be able to group tabs together and create a new folder (or group tab name, don't really know the name that it's called) as the default instead of grouping them to a parent tab.

I believe edge's vertical tab groupping can do something similar to this. I just don't know if there is an option I am missing for side berry or it cannot do this.

**Comment by llc0930** [comment]
Select the desired tab, right-click to open the context menu, and click "Group".
Or use a custom shortcut key (if you have defined it in the settings page).
